packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -1357,8 +1362,8 @@ def process_payload(self, payload: BaseTxPayload) -> None:
                 f"sender {sender} has already sent value for round: {self.round_id}"
             )
 
-        if self._hash_length:
-            content = payload.data.get(self.payload_attribute)
+        if getattr(self, "_hash_length", None):
+            content = "".join(payload.tx_hashes)
```

Comment:
temporary to accomodate the changes, this logic will be moved from here to elsewhere later